radixdlt-java/src/main/java/com/radixdlt/client/core/address/RadixUniverseConfig.java

```diff
@@ -25,7 +31,17 @@ public static RadixUniverseConfig fromDson(String universeDson) {
 		RadixUniverseConfig universe = RadixJson.getGson().fromJson(jsonElement, RadixUniverseConfig.class);
 		return universe;
 	}
+	
+	public static RadixUniverseConfig fromFile(File file){
```

Comment:
Instead of dealing with the file use the stream.

```
public static RadixUniverseConfig fromInputStream(InputStream inputStream) {
		try {
			return RadixJson.getGson().fromJson(new InputStreamReader(inputStream), RadixUniverseConfig.class);
		} catch (Exception e) {
			throw new IllegalStateException("Failed to read stream from bootstrap file");
		}
	}
```